From 10a36604ad136710ed64da94c551c32f2695cd1f Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Thu, 1 May 2025 21:18:27 +0200
Subject: [PATCH 058/116] dt-bindings: net: pcs: mediatek,sgmiisys: add phys
 and resets

For mt7988 we need 2 additional properties:

- phys
- resets

Signed-off-by: Frank Wunderlich <frank-w@public-files.de>
---
 .../bindings/net/pcs/mediatek,sgmiisys.yaml       | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/Documentation/devicetree/bindings/net/pcs/mediatek,sgmiisys.yaml b/Documentation/devicetree/bindings/net/pcs/mediatek,sgmiisys.yaml
index 1bacc0eeff75..ca0f4f4ec765 100644
--- a/Documentation/devicetree/bindings/net/pcs/mediatek,sgmiisys.yaml
+++ b/Documentation/devicetree/bindings/net/pcs/mediatek,sgmiisys.yaml
@@ -58,6 +58,11 @@ properties:
           - const: sgmii_tx
           - const: sgmii_rx
 
+      phys:
+        $ref: /schemas/types.yaml#/definitions/phandle
+        description:
+          Phandle to the (xfi) phy.
+
     required:
       - compatible
       - clocks
@@ -65,6 +70,16 @@ properties:
 
     additionalProperties: false
 
+  resets:
+    description:
+      A phandle to reset the sgmiisys controller.
+    items:
+      - items:
+          - description:
+              Phandle to the reset controller
+          - description:
+              value for reset
+
 required:
   - compatible
   - reg
-- 
2.39.5

